models/social/arbitrum/social_arbitrum_trades.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+    schema = 'social_arbitrum',
+    tags = ['dunesql'],
+    alias = alias('trades'),
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash','evt_index']
+    )
+}}
+
+{% set base_models = [
+    (ref('cipher_arbitrum_base_trades'))
+    , (ref('post_tech_arbitrum_base_trades'))
+] %}
+
+WITH trades AS (
+    {{enrich_social_trades('arbitrum', base_models, source('arbitrum', 'transactions'))}}
```

Comment:
we shouldn't enrich quite yet, that should be saved for `social_trades` final spell

Reply:
applies to all blockchains

Reply:
Unlike dex.trades which only uses crosschain spells to enrich, this one uses the .transactions table for every chain. So given the specific enrichment inputs for each chain anyways, doesn't it make sense to enrich at a chain level here?